performance-tests/docker-compose.yaml

```diff
@@ -42,7 +42,7 @@ services:
       retries: 5
       timeout: 10s
   kroxylicious:
-    image: ${KROXYLICIOUS_IMAGE:-quay.io/kroxylicious/kroxylicious:0.5.0-SNAPSHOT}
+    image: ${KROXYLICIOUS_IMAGE:-quay.io/kroxylicious/kroxylicious:0.6.0-SNAPSHOT}
```

Comment:
This is a defect in the release process

https://github.com/kroxylicious/kroxylicious/issues/1093